grails-app/views/species/search.gsp

```diff
@@ -28,7 +28,8 @@
     <asset:script type="text/javascript">
         // global var to pass GSP vars into JS file
         SEARCH_CONF = {
-            searchResultTotal: ${searchResults.totalRecords},
+            searchResultTotal: ${searchResults.totalRecords},      
+            searchSpecieEnable: "${grailsApplication.config.searchSpecie.enable}",
```

Comment:
This should be added to `Application.groovy` with a default value (true) to not break things.